previous-version/README.md

```diff
@@ -0,0 +1,40 @@
+# previous-version
+
+Given a release version identifies release type and a corresponding previous release version
+
+## Usage
+
+This GHA can be used in any repository, but requires it to be fetched with full history,
+so that existing tags can be analyzed. Otherwise, results may be incorrect.
+
```

Comment:
Let's also add similar table for outputs. Inputs + Outputs would fully describe the interface of this action.